### PR TITLE
Fix bug with flashcard images

### PR DESCRIPTION
**Pull request by Froggerson**
Previously, flashcards would display alternative text when no images were passed in as props, which was not really ideal.
Before:
image
After:
Screenshot 2020-08-10 at 9 02 02 AM

### CODE REVIEW

File: frontend/src/flashcards/FlashcardFront.js [hunk 1]
```diff
@@ -16,35 +16,40 @@ const frontStyle={
...
+            src={this.props.image}
+            alt="Can not fetch image"
+          />
+        ) : null}
+        {this.props.text}
+      </div>
+    );
+  }
+}
 /*<img src={this.props.image}/>*/
```
Comment (ethyus) [inline frontend/src/flashcards/FlashcardFront.js]: Is there a particular need for keeping this comment?

Comment (Froggerson) [reply frontend/src/flashcards/FlashcardFront.js]: No. I think I'll go ahead and delete it.

Comment (Froggerson) [reply frontend/src/flashcards/FlashcardFront.js]: Ok I deleted it!

**Review by zmr — APPROVED** [review]

File: frontend/src/flashcards/FlashcardFront.js [hunk 2]
```diff
@@ -16,35 +16,39 @@ const frontStyle={
...
-  maxHeight:"150px",
-  width:"auto",
+  maxHeight: "150px",
+  width: "auto",
   height: "auto",
-  margin:"10px",
+  margin: "10px",
 };
 
 class FlashcardFront extends React.Component {
```
Comment (tylerhou) [inline frontend/src/flashcards/FlashcardFront.js]: This component has no state, so you can use a function component instead: https://reactjs.org/docs/components-and-props.html#function-and-class-components.

If you use a function component, it's easier for a code reader to immediately understand that the function has no state.

**Review by tylerhou — APPROVED** [review]

**Pull request merged**